### PR TITLE
Minimap

### PR DESCRIPTION
**Pull request by eordano**
Implements a first version of the minimap

image

### CODE REVIEW

File: webapp/src/components/Minimap.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './Minimap.css'
+
+const PARCELS = 500
```
Comment (nicosantangelo) [inline webapp/src/components/Minimap.js]: Both this and the `250` below are related to the max number of parcels? if so, `parcelUtils` has a `getBounds()` method used by the rest of the codebase to attest how many parcels are there.

What I mean is that I'd extract it **if** this two numbers are related to some other part of the codebase to simplify future changes

File: webapp/src/containers/MinimapContainer.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      minX: PropTypes.number,
+      minY: PropTypes.number,
+      maxX: PropTypes.number,
+      maxY: PropTypes.number
+    })
+  }
+
+  constructor(...args) {
+    super(...args)
+    this.onDragEnd = newRange => ({})
```
Comment (nicosantangelo) [inline webapp/src/containers/MinimapContainer.js]: What's this? and why not a class method?

**Comment by nicosantangelo** [comment]
### Some (minor) comments:

* The minimap is visible on loading, while the rest of the UI isn't.
image
* 

### Style
* What do you think of making the mini-square a bordered rectangle instead of a darker one?, like this:
image

---

Looks really cool!

File: webapp/src/containers/MinimapContainer.js (hunk 2 above)
Comment (eordano) [reply webapp/src/containers/MinimapContainer.js]: Hey, sorry, this should be out. I left it there to implement click+drag to move around, but never got to it.

**Comment by eordano** [comment]
Implemented all of the changes

**Comment by nicosantangelo** [comment]
🎉 🌮

**Pull request merged**